mne/report.py

```diff
@@ -259,6 +263,12 @@ def _update_html(html, report_fname, report_sectionlabel):
                 report_fname = fname
                 report_sectionlabel = 'raw'
             elif fname.endswith(('-fwd.fif', '-fwd.fif.gz')):
+                if report.subjects_dir is not None:
+                    html = report._render_sensitivity_map(fname, report.subjects_dir)
+                    report_fname = fname + '(sensitivity)'
```

Comment:
I do not think that we should change the filename. There is no way to guess what's in a report. We should not start doing this. Let's leave it to the users to choose their names.